src/gamma_inc.jl

```diff
@@ -781,7 +781,9 @@ gamma_inc(a::Real,x::Real,ind::Integer=0) = _gamma_inc(promote(float(a),float(x)
 function _gamma_inc(a::Float64, x::Float64, ind::Integer)
     iop = ind + 1
     acc = acc0[iop]
-    if a < 0.0 || x < 0.0
+    if isnan(a) || isnan(x)
+        return (a*x, a*x)
```

Comment:
Isn't this always
```suggestion
        return (NaN, NaN)
```
? And should it maybe be the second branch such that `gamma_inc(-1.0, NaN)` etc. still throws an error?

Reply:
The idea was to propagate the same NaN in case somebody wanted to check if it was a
```julia
julia> reinterpret(UInt64, 0.0/0.0)
0xfff8000000000000
instead of a
```julia
julia> reinterpret(UInt64, NaN)
0x7ff8000000000000
```
I agree that it's good to have it after the negativity checks so I'll change taht.